### PR TITLE
Updated Microsoft browser extension policy URL

### PR DESCRIPTION
**Pull request by MaxSteinMS**
Current URL: https://docs.microsoft.com/legal/windows/agreements/microsoft-browser-extension-policy leads to 404.
Updated URL to reflect new URL: https://docs.microsoft.com/legal/microsoft-edge/microsoft-browser-extension-policy.

### CODE REVIEW

File: browsers/edge/shortdesc/microsoft-browser-extension-policy-shortdesc.md [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
-audience: itpro
manager: dansimp
+audience: itpro
+manager: dansimp
 ms.prod: edge
 ms.topic: include
 ---
 
-[Microsoft browser extension policy](https://docs.microsoft.com/legal/windows/agreements/microsoft-browser-extension-policy):
+[Microsoft browser extension policy](https://docs.microsoft.com/legal/microsoft-edge/microsoft-browser-extension-policy:
```
Comment (illfated) [inline browsers/edge/shortdesc/microsoft-browser-extension-policy-shortdesc.md:12]: The link is duly corrected now. Just need to keep the closing bracket to make the link text appear as clickable text (and hide the URL):
```suggestion
[Microsoft browser extension policy](https://docs.microsoft.com/legal/microsoft-edge/microsoft-browser-extension-policy):
```

**Comment by denisebmsft** [comment]
And thank you @illfated

**Comment by illfated** [comment]
Sorry to see that you did not want to keep the working link text formatting.

**Comment by illfated** [comment]
Unfortunately, my suggestion was not committed, it was only marked as "already resolved". Do you want to wait until someone complains about the weird URL + text starting with a [ bracket ?

**Comment by e0i** [comment]
@illfated It seems the PR has already been merged with no Markdown formatting errors, and with your suggestions:

https://github.com/MicrosoftDocs/windows-itpro-docs/pull/6561/files?short_path=99ad82d#diff-99ad82d68eeb3f6497474786bf867307

**Comment by illfated** [comment]
You are completely right. I should have checked the commit itself (https://github.com/MicrosoftDocs/windows-itpro-docs/commit/9d76ba0380b00f55b0b4c3eeb9e7c7f1385fe722) to see that it was added during the merge process.

**Pull request merged**